### DISCUSSION TITLE
R 5.0.0 Scope mismatch added in Microsite Release details and in the Sign off provided for Samagra Prashnavali

### DISCUSSION THREAD

**Discussion by anunagaraj** [Sunbird-coKreat/Community#20] [General Discussions]
@pallakartheekreddy @surendrasinghs @tarang27 @ShailandraOjha @luckynvdu @sharathkashyap @chitranshu-keshav

**Tickets added in Microsite for Samagra** 
![image](https://user-images.githubusercontent.com/40622169/188598691-7f257ac1-6ed7-4c37-a649-316476ca6a41.png)

**Tickets signed off** 
![image](https://user-images.githubusercontent.com/40622169/188598789-6ad4fe7d-7ea0-4bc5-be82-7a4b0e51ee80.png)

The below ticket is not mentioned in the sign off  shared by Samagra team - https://project-sunbird.atlassian.net/browse/CO-18. Please update the Microsite release details as per the sign off provided

**Comment by tarang27** [comment]
Updated the sign off sheet

**Reply by ShailandraOjha** [reply]
Hi @tarang27, We will need to ensure no discrepancy in the scope signed off and the Scope updated in microsites.
cc: @chitranshu-keshav @surendrasinghs @luckynvdu
